plugins/shared/annotate/index.js

```diff
@@ -20,6 +20,9 @@ require("./style.less");
 // and across.
 const MIN_HIGHLIGHT_SIZE = 25;
 
+// typecast to jQuery collection in case our plugin is jquery-less
+const ensureJqueryCollection = (el) => (el instanceof $) ? el : $(el);
```

Comment:
Needed to add this since the Focus Tracker doesn't use jQuery and our annotations rely on receiving a jQuery collection.

Reply:
Yeah, I'd love to get rid of jQuery in tota11y at some point.